packages/graphback/src/layers/data/PgKnexDBDataProvider.ts

```diff
@@ -0,0 +1,68 @@
+// tslint:disable-next-line: no-require-imports
+import * as Knex from 'knex';
+import { AdvancedFilter, GraphbackDataProvider } from './GraphbackDataProvider';
+import { NoDataError } from './NoDataError';
+ 
+/**
+ * Knex.js database data provider exposing basic CRUD operations. 
+ */
+// tslint:disable-next-line: no-any
+export class PgKnexDBDataProvider<Type = any, GraphbackContext = any> implements GraphbackDataProvider<Type, GraphbackContext>{
```

Comment:
@wtrocki why don't have just a Knex resolver which will be compatible with SQLite and Postgres?

Reply:
Because there compatibility is theoretical. There will be difference between create method. Most DB do not have ability to return all data on create but postgress has. I  will simply extend this class and change this small element.

Reply:
Another reason is the actual dependency to specific db. If we split to packages we do not want them to bring extra dependency thst will not be used

Reply:
I see than it's better to have a dedicated one for PG, but with can still have a generic one from SQLite (MariaDB could also enter in the generic one)